whelktool/scripts/syslib/Copy_holdings.groovy

```diff
@@ -0,0 +1,45 @@
+String CURRENT_SIGEL = 'https://libris.kb.se/library/Ssb'
+String NEW_SIGEL = 'https://libris.kb.se/library/Mlc'
+
+File bibIDs = new File(scriptDir, "Copy_hold_bibID.txt")
+
+def holdList = []
+
+String bibidstring = bibIDs.readLines().join("','")
+
+selectBySqlWhere("""id in
+
+(
+ select lh.id
+  from
+   lddb lb
+  left join
+   lddb lh
+  on lh.data#>>'{@graph,1,itemOf,@id}' = lb.data#>>'{@graph,1,@id}'
+ where lb.data#>>'{@graph,0,controlNumber}' in ( '$bibidstring' ) and lb.collection = 'bib'
+ and
+ lh.data#>>'{@graph,1,heldBy,@id}' = '{$CURRENT_SIGEL}'
```

Comment:
This might be correct, but it looks off to me. Shouldn't it be:
either
${CURRENT_SIGEL}
or
$CURRENT_SIGEL
?

Reply:
That is a typo, I'll fix it. Interesting that it still seems to work. =)